packages/config/test/options.test.js

```diff
@@ -237,6 +237,13 @@ describe('config: options', () => {
       expect(consola.warn).toHaveBeenCalledWith('build.extractCSS.allChunks has no effect from v2.0.0. Please use build.optimization.splitChunks settings instead.')
     })
   })
+
+  describe('config: loading screen', () => {
+    test('should add loading screen', () => {
+      const config = getNuxtConfig({ dev: true })
+      expect(config.devModules).toStrictEqual(['@nuxt/loading-screen'])
```

Comment:
Would it be possible to use [expect.arrayContaining](https://jestjs.io/docs/en/expect#expectarraycontainingarray) here? Then if we add a second devModule this test will still succeed.